<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command &amp; Control</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Icons/icon3_7_1.png</normaloff>../../Icons/icon3_7_1.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab:disabled
{
	width: 0px;
    min-width: 0px;
    max-width: 0px; 
    color: rgba(0,0,0,0);
    background-color: rgba(0,0,0,0);
}
</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_config">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Config</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_map">
     <attribute name="title">
      <string>Map</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_comms">
     <attribute name="title">
      <string>Comms</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_net">
     <attribute name="title">
      <string>Network</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_data">
     <attribute name="title">
      <string>Data</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_docs">
     <attribute name="title">
      <string>Documents</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_radar">
     <attribute name="title">
      <string>Radar</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSex">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTabs">
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="title">
     <string>Tabs</string>
    </property>
    <addaction name="actionConfig"/>
    <addaction name="actionMap"/>
    <addaction name="actionComms"/>
    <addaction name="actionNet"/>
    <addaction name="actionData"/>
    <addaction name="actionDocuments"/>
    <addaction name="actionRadar"/>
   </widget>
   <addaction name="menuSex"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuTabs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConfig">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/gear-dark.png</normaloff>
     <normalon>:/Icons/gear-light.png</normalon>:/Icons/gear-dark.png</iconset>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="actionMap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/map-dark.png</normaloff>
     <normalon>:/Icons/map-light.png</normalon>:/Icons/map-dark.png</iconset>
   </property>
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionComms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/comm-dark.png</normaloff>
     <normalon>:/Icons/comm-light.png</normalon>:/Icons/comm-dark.png</iconset>
   </property>
   <property name="text">
    <string>Comms</string>
   </property>
  </action>
  <action name="actionNet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/net-dark.png</normaloff>
     <normalon>:/Icons/net-light.png</normalon>:/Icons/net-dark.png</iconset>
   </property>
   <property name="text">
    <string>Net</string>
   </property>
  </action>
  <action name="actionData">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/db-dark.png</normaloff>
     <normalon>:/Icons/db-light.png</normalon>:/Icons/db-dark.png</iconset>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </action>
  <action name="actionDocuments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/docs-dark.png</normaloff>
     <normalon>:/Icons/docs-light.png</normalon>:/Icons/docs-dark.png</iconset>
   </property>
   <property name="text">
    <string>Documents</string>
   </property>
  </action>
  <action name="actionRadar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/radar-dark.png</normaloff>
     <normalon>:/Icons/radar-light.png</normalon>:/Icons/radar-dark.png</iconset>
   </property>
   <property name="text">
    <string>Radar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
